stl/inc/chrono

```diff
@@ -619,8 +652,27 @@ namespace chrono {
             // but the initial value of _Ctr could be large.
             const long long _Whole = (_Ctr / _Freq) * period::den;
             const long long _Part  = (_Ctr % _Freq) * period::den / _Freq;
-            return time_point(duration(_Whole + _Part));
+            return _Whole + _Part;
+        }
+
+#if (defined(_M_IX86) || defined(_M_X64) || defined(_M_ARM) || defined(_M_ARM64)) && !defined(_M_CEE_PURE)
+        _NODISCARD static long long _Atomic_load_ll_relaxed(volatile long long* _Mem) noexcept {

```

Comment:
Should these two `_Atomic_meow` functions live in `<xatomic.h>` so they'll stay in sync with the rest of the atomic code?

Reply:
They can be moved, but implementation still won't be shared, as `<atomic>` has memory order as run-time constant, and these functions are simplified by having memory order substituted. 

Still it may be helpful to move to avoid missing updating if `<atomic>` is updated.

Ultimately I think that properly sized atomics `_Atomic_storage<_Ty, 1>`...`_Atomic_storage<_Ty, 8>`, `_Atomic_integral<_Ty, 1>`...`_Atomic_integral<_Ty, 8>` should be in an internal header. Whereas non-essential sugar, such as generic `atomic<T>`, `atomic_flag` and free functions should stay in `<atomic>`.

I expect that internal atomics of proper size are useful for patterns like this, for `latch` / `sempathore` / `barrier` #52, for even better `call_once` #707, possbily for anything else. 

I see the atomic movement can be done after #694 to avoid #694 re-doing.